x-pack/plugins/fleet/common/services/agent_status.ts

```diff
@@ -49,7 +49,7 @@ export function buildKueryForUnenrollingAgents() {
 }
 
 export function buildKueryForOnlineAgents() {
-  return `not (${buildKueryForOfflineAgents()}) AND not (${buildKueryForErrorAgents()}) AND not (${buildKueryForEnrollingAgents()}) AND not (${buildKueryForUnenrollingAgents()})`;
+  return `not (${buildKueryForOfflineAgents()}) AND not (${buildKueryForErrorAgents()}) AND not (${buildKueryForUpdatingAgents()})`;
```

Comment:
it probably doesn't make much performance difference but none of these `buildKuery` methods need to be methods. they can just be constants

Reply:
the `buildKueryForOfflineAgents` need to be a method as we use the current date inside, so for consistency I think we should keep all of them methods.